### ISSUE TITLE
Link is pointing to the wrong phone number

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#213]
Screengrab/videos (insert file names here)
2021 06 03, Link has wrong name.png

Link
https://www.boston.gov/departments/emergency-medical-services/cpr-training

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 11.4 > Chrome 91

Trusted Tester Test
Links

Brief Description
There is a link on the page that has a wrong name.

Steps to reproduce
1)	Go to the URL
2)	Go down to Step 3 in the CPR training guide
3)	Look at the link named 617-343-1027 and inspect the link
a)	Bug: The link has a href that does not match the name of the link.  A different phone number is listed with the last 4 digits of “1029” rather than “1027”

Desired Result
•	Links on the page have hrefs that matches where the link is supposed to go to

Actual Result
•	There is a link on the page that does not have a href that matches where the link is supposed to go to.

**Comment by david-iterators** [comment]
![2021 06 03, Link has wrong name](https://user-images.githubusercontent.com/33845414/120938285-eeb86280-c6df-11eb-96f4-8f88c8652a3b.png)

**Comment by subaha-cob** [comment]
@david-iterators This is all set on `Production`.

**Comment by ollie-iterators** [comment]
This has been fixed
